From f384518dedec4ad657ad69d55754076c13c22d96 Mon Sep 17 00:00:00 2001
From: Johan Oudinet <johan.oudinet@gmail.com>
Date: Wed, 20 Jan 2016 08:11:32 -0500
Subject: [PATCH] remove checking erlang version

Without this patch, the configure will try to run erlang to simply check
if the version is supported by ejabberd. Instead, we do this test
statically.

Signed-off-by: Johan Oudinet <johan.oudinet@gmail.com>
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 97e89a2..cd1ab08 100644
--- a/configure.ac
+++ b/configure.ac
@@ -39,10 +39,8 @@ AC_ARG_ENABLE(erlang-version-check,
 	[Check Erlang/OTP version @<:@default=yes@:>@])])
 case "$enable_erlang_version_check" in
 	yes|'')
-		ERLANG_VERSION_CHECK([$REQUIRE_ERLANG_MIN],[$REQUIRE_ERLANG_MAX])
 		;;
 	no)
-		ERLANG_VERSION_CHECK([$REQUIRE_ERLANG_MIN],[$REQUIRE_ERLANG_MAX],[warn])
 		;;
 esac
 
-- 
2.5.0

